packages/components/src/Account/Auth/UserSetAvatar.tsx

```diff
@@ -135,12 +143,14 @@ export class UserSetAvatar extends React.Component<UserSetAvatarAuthProps, UserS
                       size={buttonSizes.SMALL}
                       textTransform={"none"}
                       onClick={() => this.setState({ image: undefined })}
+                      disabled={this.state.saveInProgress}
                     >
                       Go back
                     </InvertedButton>
                     <SaveButtonContainer>
-                      <Button size={buttonSizes.SMALL} textTransform={"none"} type={"submit"}>
-                        Save
+                      <Button size={buttonSizes.SMALL} textTransform={"none"} type={"submit"} disabled={this.state.saveInProgress}>
+                        {this.state.saveInProgress && <>Saving <ClipLoader size={16} /></>}
```

Comment:
Low priority but FYI I recently added a `loading` prop to `Button` that, when true, replaces its children with `ClipLoader`, for uses like this.